data/libs/Equipment.lua

```diff
@@ -469,159 +469,161 @@ end
 -- autopilot - autopilot
 -- trade_computer - commodity trade analyzer computer module
 
+local CARGOLANGRESOURCE = "cargo"
```

Comment:
I'm not sure I understand why this has been parameterized?

Reply:
because splitting it to a separate lang file named by me might not be in line with everyone elses ideas of consisten file naming. so instead of changing X number of entries I made 1 reference to the filename and the rest reference that "constant"